src/Filters/Has.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace LaravelJsonApi\Eloquent\Filters;
+
+use LaravelJsonApi\Eloquent\Contracts\Filter;
+use LaravelJsonApi\Eloquent\Schema;
+
+class Has implements Filter
+{
+    use Concerns\DeserializesValue;
```

Comment:
This filter only supports the value being a boolean, so we don't need to use `DeserializesValue` here. Please remove.